src/main/java/frc/trigon/robot/subsystems/swerve/Swerve.java

```diff
@@ -231,6 +236,13 @@ void selfRelativeDrive(double xPower, double yPower, MirrorableRotation2d target
         selfRelativeDrive(speeds);
     }
 
+    private Optional<Rotation2d> getRotationTargetOverride() {
+        if (!CameraConstants.NOTE_DETECTION_CAMERA.hasTargets())
```

Comment:
You want to always override the angle? When will you override for the speaker anlge?